spec/support/stripe.rb

```diff
@@ -0,0 +1,8 @@
+require 'stripe'
+module Stripe
+  module CertificateBlacklist
+    def self.check_ssl_cert(uri, ca_file)
```

Comment:
Unused method argument - `uri`. If it's necessary, use `_` or `_uri` as an argument name to indicate that it won't be used. You can also write as `check_ssl_cert(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `ca_file`. If it's necessary, use `_` or `_ca_file` as an argument name to indicate that it won't be used. You can also write as `check_ssl_cert(*)` if you want the method to accept any arguments but don't care about them.